src/_locales/en_US/messages.json

```diff
@@ -23,6 +23,10 @@
         "message": "Currently Privacy Badger only checks if third parties are using cookies, HTML5 local storage, or canvas fingerprinting to track your browsing. Some of these domains may be using tracking methods that Privacy Badger can't detect.",
         "description": ""
     },
+    "options_enable_dnt_checkbox": {
+        "message": "Enable sending DNT header",
+        "description": ""
```

Comment:
Could you add a description to provide translators with context? You could describe where in Badger's UI this label is located.

Reply:
Done